ChangeLog.txt

```diff
@@ -1,8 +1,31 @@
 Next v0.1.X
 -----------
 
+Cultures:
+
+- Added namelist for Vandal culture
+- Added dynasty names for Roman, Romano-Gallic, Romano-Hispanic, Romano-Thracian, Romano-Illyrian and Romano-Aramean cultures
+- Added many new name equivalencies and fixed some existing ones
+
+Localization:
+- Added numerous cultural title localization for Britannia, Gallia, Italia, Hispania, Africa, Eastern Roman Empire, Persia, Pontic-Caspian steppe, Tamazgha and Germania Magna
+- Fixed some typos among the title names
+
+History:
+- Expanded Nepos' Dalmatia based on a source, which states that it included lands between Drina and Sava. In addition, Dalmatia is now controls Epidaurum instead of ERE
```

Comment:
Which source is this?

Reply:
> Owner

Mostly these ones(https://www.academia.edu/691311/Tracing_the_Language_of_the_Vandals, https://www.academia.edu/1516556/THE_LANGUAGE_AND_NAMES_OF_THE_VANDALS).

Reply:
> Which source is this?

https://www.jstor.org/stable/4435771

Reply:
That is not what the source says, it explicitly states that Theoderic crossed the Sava southwards in the 470s and established control over the area.

Reply:
> That is not what the source says, it explicitly states that Theoderic crossed the Sava southwards in the 470s and established control over the area.

Fine, I've reverted my change, with the exception of Ragusa part.